resources/views/widgets/graph.blade.php

```diff
@@ -1,9 +1,13 @@
 <div class="dashboard-graph">
-    <a href="graphs/{{ $param }}/type={{ $graph_type }}/from={{ $from }}">
+    <a href="graphs/
+@foreach($params as $param){{ $param }}/@endforeach
```

Comment:
Wouldn't implode be nicer than foreach?
`{{ implode($params, '/') }}`

Reply:
Thanks. I didn't know you can utilize PHP functions in Blade like that.